src/components/dataPanel/PanelViewState.tsx

```diff
@@ -27,6 +27,7 @@ export const PanelViewState: FunctionComponent<IPanelViewStateProps> = ({
   columns = 1,
   splitAfter = Math.ceil(fields.length / 2),
 }) => {
+  columns = splitAfter ? 2 : columns;
```

Comment:
If user of `<PanelViewState>` sets the `splitAfter` prop, automatically set the number of columns to 2. (Tripped myself up by forgetting to set `columns` to 2)